src/lib/build/config.ts

```diff
@@ -18,6 +18,21 @@ const getConfig = (path = "./config/canopy.json", isDev = false) => {
   const defaultConfig = JSON.parse(defaultData);
   const config = data && JSON.parse(data);
 
+  /**
```

Comment:
This fixes an omission introduced over the weekend where the default canopy config does not have a `label` or `summary`, making it so our example site has the label "IIIF Collection" which is the `label` on the source collection.

![image](https://github.com/canopy-iiif/canopy-iiif/assets/7376450/f0125ba2-e28b-4403-9e08-516bf2d8eff4)